### PR TITLE
Document that Debian 8 and Ubuntu 14 systemd

### LINKED ISSUE #13865: elasticsearch service not starting on Debian 8.2 boot
Elasticsearch 1.7.2 is installed (using the Debian repo you guys provide) and works properly. When I run `sudo service elasticsearch start`, it starts up and runs as expected. I've set it up to start on boot by issuing `sudo update-rc.d elasticsearch defaults 95 10`. `elasticsearch` exists in both `/etc/init.d` and `/etc/rc[0-6].d`. Every time I boot my server, though, it doesn't start; I have to boot it manually. The only thing that exists in `/var/log/elasticsearch/elasticsearch.log` is this:

```
[2015-09-29 17:27:10,696][INFO ][node                     ] [Changeling] stopping ...
[2015-09-29 17:27:10,748][INFO ][node                     ] [Changeling] stopped
[2015-09-29 17:27:10,748][INFO ][node                     ] [Changeling] closing ...
[2015-09-29 17:27:10,754][INFO ][node                     ] [Changeling] closed
```

What might be causing this? I'm not sure where to look. I've grepped the log files in `/var/log/` without success.

### PR DESCRIPTION
**Pull request by kkirsche**
Fix #13865

### CODE REVIEW

File: docs/reference/setup/as-a-service.asciidoc [hunk 1]
```diff
@@ -73,7 +75,7 @@ sudo service elasticsearch start
 [float]
```
Comment (clintongormley) [inline docs/reference/setup/as-a-service.asciidoc]: Add an ID:

    [[using-systemd]]
    [float]
    ....

File: docs/reference/setup/as-a-service.asciidoc [hunk 2]
```diff
@@ -39,6 +39,8 @@ sudo update-rc.d elasticsearch defaults 95 10
 sudo /etc/init.d/elasticsearch start
 --------------------------------------------------
 
+Users running Debian 8 or Ubuntu 14 or later may require configuration of systemd instead of `update-rc.d`. In those cases, please refer to the https://www.elastic.co/guide/en/elasticsearch/reference/current/setup-service.html#_using_systemd[Using `systemd`] section.
```
Comment (clintongormley) [inline docs/reference/setup/as-a-service.asciidoc]: please refer to <<using-systemd>>.

**Comment by clintongormley** [comment]
Hi @kkirsche 

Small change to use an internal link instead of an http link

thanks

**Comment by kkirsche** [comment]
@clintongormley fixed — thanks for sharing how to make that change

**Comment by clintongormley** [comment]
Thanks @kkirsche - merged

**Comment by kkirsche** [comment]
My pleasure.

**Pull request merged**